docs/readme.md

```diff
@@ -39,13 +41,31 @@ Many modules consuming Lighthouse are only interested in the performance numbers
 You can limit the audits you run to a particular category or set of audits.
 
 ```js
-const perfConfig = require('lighthouse/lighthouse-core/config/perf-config.js');
 // ...
-launchChromeAndRunLighthouse(url, flags, perfConfig).then( // ...
+const flags = {onlyCategories: ['performance']};
+launchChromeAndRunLighthouse(url, flags).then( // ...
 ```
 
 You can also craft your own config (e.g. [mixed-content-config.js](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/mixed-content-config.js)) for custom runs. Also see the [basic custom audit recipe](https://github.com/GoogleChrome/lighthouse/tree/master/docs/recipes/custom-audit).
 
+### A warning about flags
```

Comment:
maybe a less ominous title for the section? :) Just `### CLI flags that have no effect as module options` or something better than that

Reply:
went with "Differences from CLI flags"